apis/apiextensions/v1/conditions.go

```diff
@@ -48,6 +53,9 @@ const (
 
 	ReasonValidPipeline       xpv1.ConditionReason = "ValidPipeline"
 	ReasonMissingCapabilities xpv1.ConditionReason = "MissingCapabilities"
+
+	ReasonWatchCircuitOpen   xpv1.ConditionReason = "WatchCircuitOpen"
+	ReasonWatchCircuitClosed xpv1.ConditionReason = "WatchCircuitClosed"
```

Comment:
I found myself getting confused about the meaning of open/closed a few times. It makes total sense from an electrical circuit / breaker standpoint, but sometimes when I saw `Closed` my mind went to "closed for business", i.e. the opposite of what it really means.

I'm wondering if this terminology is going to be confusing for end users. What do you think about alternatives, like `Paused`, `Halted`, `Stopped`, etc.? 🤔

Reply:
Thanks @negz - I don't think any alternatives are clearly better, and there does appear to be some prior art here with Istio, so it's reasonable to go with what you've already done. The descriptive message also helps a lot here.